bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_TypeI_DiTau/WGammaFusion/baseDatacards/WGammaFusion_thirdParZZZ_secondParYYY_firstParXXX_run_card.dat

```diff
@@ -0,0 +1,304 @@
+#***********************************************************************
+# Renormalization and factorization scales (advanced and NLO options)  *
+#***********************************************************************
+  False	=  fixed_QES_scale   ! if .true. use fixed Ellis-Sexton scale
+  91.118	=  QES_ref_fixed     ! fixed Ellis-Sexton reference scale
+  1.0	=  muR_over_ref      ! ratio of current muR over reference muR
+  1.0	=  muF_over_ref     ! ratio of current muF over reference muF
+  1.0	=  QES_over_ref      ! ratio of current QES over reference QES
+#*********************************************************************** 
+# Reweight flags to get scale dependence and PDF uncertainty           *
+# For scale dependence: factor rw_scale_up/down around central scale   *
+# For PDF uncertainty: use LHAPDF with supported set                   *
+#***********************************************************************
+  True	=  reweight_scale    ! reweight to get scale dependence
+  1.0, 2.0, 0.5 = rw_rscale ! muR factors to be included by reweighting
+  1.0, 2.0, 0.5 = rw_fscale ! muF factors to be included by reweighting
+#***********************************************************************
+#***********************************************************************
+# Store reweight information in the LHE file for off-line model-       *
+# parameter reweighting at NLO+PS accuracy                             *
+#***********************************************************************
+ True = store_rwgt_info ! Store info for reweighting in LHE file
+#***********************************************************************
+# ickkw parameter:                                                     *
+#   0: No merging                                                      *
+#   3: FxFx Merging - WARNING! Applies merging only at the hard-event  *
+#      level. After showering an MLM-type merging should be applied as *
+#      well. See http://amcatnlo.cern.ch/FxFx_merging.htm for details. *
+#   4: UNLOPS merging (with pythia8 only). No interface from within    *
+#      MG5_aMC available, but available in Pythia8.                    *
+#  -1: NNLL+NLO jet-veto computation. See arxiv:1412.8408 [hep-ph].    *
+#***********************************************************************
+  0	=  ickkw
+ #***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma)                                       *
+#***********************************************************************
+  15.0	=  bwcutoff
+#**********************************************************
+# Apply pt/E/eta/dr/mij cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#**********************************************************
+   F  = cut_decays    ! Cut decay products
```

Comment:
if you want the cuts following to be applied on the jets from n3 decay, you need to use "True = cut_decays" here